ironfish-cli/src/commands/wallet/rescan.ts

```diff
@@ -28,14 +28,11 @@ export class RescanCommand extends IronfishCommand {
       description: 'Sequence to start account rescan from',
       hidden: true,
     }),
-    full: Flags.boolean({
```

Comment:
This flag is deleted in favor of wallet/reset which should be used. Rescan should never delete wallet data like the birthday, only cause a chain rescan.

Reply:
I think we'll want a `reset` CLI command as part of this, so multisig users can easily access it -- maybe it'll be easier to pull it out into a CLI command then

Reply:
That sounds good, want me to add the reset command to this PR?

Reply:
I think it should either be in this PR, or we should pull the removal out into a separate PR, so we're not potentially leaving staging in a state where doing a release would leave a gap in functionality. You could pull the removal out into a separate PR and just temporarily shim it in this PR with `resetAccount({ resetCreatedAt: true })`

Reply:
I'll create the `ironfish wallet:reset` command in this PR.